lib/gettext-setup/merge_pot.rb

```diff
@@ -0,0 +1,9 @@
+# A task to merge pot files located within the locales directory into one
+# pot file for localization. Output is 'completed.pot' in locales directory.
+module GettextSetup
+  module MergePot
+    def self.merge_pot
+      `msgcat locales/*.pot > locales/completed.pot`
```

Comment:
If this command fails, what happens with the rake task/its output? In similar tasks that need to shell out, we've been using `system` and checking `exitstatus`, or using `Open3.popen3` and checking the resulting `wait.value`. Ideally, the task should `exit 1` if the command exits with 1.